cluster/config.go

```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/minio/minio/pkg/env"
 )
 
+func getNs() string {
+	return "default"
```

Comment:
are we planning to make this configurable from an env variable? or what was the reason of this function?

Reply:
yes, this is still work in progress